framework/tlvf/src/src/CmduMessageRx.cpp

```diff
@@ -7,6 +7,45 @@
  */
 
 #include <tlvf/CmduMessageRx.h>
+#include <tlvf/ieee_1905_1/tlvEndOfMessage.h>
```

Comment:
Same here, all of these includes don't belong here.
Same external class solution I suggested would solve this as well.

Reply:
Done